sceptre/cli/launch.py

```diff
@@ -36,11 +44,13 @@ def launch_command(ctx: Context, path: str, yes: bool, prune: bool):
     * Any stacks that already exist will be updated (if there are any changes)
     * If any stacks are marked with "ignore: True", those stacks will neither be created nor updated
     * If any stacks are marked with "obsolete: True", those stacks will neither be created nor updated.
-      Furthermore, if the "-p"/"--prune" flag is used, these stacks will be deleted prior to any
+    * If the "-p"/"--prune" flag is used, these stacks will be deleted prior to any
```

Comment:
No this line is a continuation of the previous point, not a new point.

Reply:
fixed